Fabric.Identity.API/scripts/Install-Identity-Windows.ps1

```diff
@@ -22,6 +22,22 @@ function Test-RegistrationComplete($authUrl)
     return $false
 }
 
+function Unlock-ConfigurationSections(){
+    $assembly = [System.Reflection.Assembly]::LoadFrom("$env:systemroot\system32\inetsrv\Microsoft.Web.Administration.dll")
```

Comment:
We already import the powershell WebAdministration module, are we able to adjust these configurations using that powershell module directly?

Reply:
ahh yeah. i think i can just move it below the import of the utilities script